nixos/modules/programs/k3b.nix

```diff
@@ -23,7 +23,7 @@
   config = lib.mkIf config.programs.k3b.enable {
 
     environment.systemPackages = with pkgs; [
-      k3b
+      kdePackages.k3b
       dvdplusrwtools
```

Comment:
I think the entire module is actually unnecessary now. We add all the things to the wrapper [here](https://github.com/NixOS/nixpkgs/blob/master/pkgs/kde/gear/k3b/default.nix#L46-L57), and then k3b elevates via kauth [here](https://invent.kde.org/multimedia/k3b/-/blob/master/src/helper/k3bhelper.cpp?ref_type=heads).